### DISCUSSION TITLE
LTS for fluentui-blazor library

### DISCUSSION THREAD

**Discussion by sebitsi** [microsoft/fluentui-blazor#2194] [General]
Hello.

First of all, congratulations on the work done so far.
Great library. I am very interested in using it.

I'm not going to ask if it's suitable for production use or not.
Somehow I already got an answer to this question.

I'm more concerned about how long this library will live.
@vnbaaij and @dvoituron are the main developers developing the library in their spare time.
They can run out of free time and stop development.
In addition, it depends on web components and its development.

Of course, one can say that it is an open source library. However, this is not a guarantee that someone with the knowledge and time will be found to develop it further.

If I understand correctly, Microsoft makes no guarantee that it will provide development.

@vnbaaij what is your opinion

Thanks for the reply.

**Comment by vnbaaij** [comment]
> I'm more concerned about how long this library will live. @vnbaaij and @dvoituron are the main developers developing the library in their spare time. They can run out of free time and stop development. 

All true. But we've been developing this for more than 3 years already and have no plans or intention to stop doing so anytime soon.

>In addition, it depends on web components and its development.

Yes, also true. But Microsoft is still heavily investing in this space. Currently a lot of work is taking place to build the next version (v3) of the web components (supporting Fluent 2 design). We are in close contact with that team and have in fact already made a few steps with implementing this new version (see the `web-components-v3` branch). It will eventually lead to a new major version of our Blazor library. 
 
> Of course, one can say that it is an open source library. However, this is not a guarantee that someone with the knowledge and time will be found to develop it further.
> 
> If I understand correctly, Microsoft makes no guarantee that it will provide development.

Again, all true. There is no guarantee made by Microsoft. However, .NET Aspire is dependent on this library for the Dashboard part. I think we can say that that alone is a sort of implicit guarantee that the library will live on for the foreseeable future.

**Reply by sebitsi** [reply]
@vnbaaij thank you for your really fast response.

I would be more reassured if Microsoft invested in this project as well. :)

Does moving the project under the Microsoft domain (github) mean that this may happen in the future?

I really don't understand how it is possible that a project like blazor doesn't have at least a basic library.
or at least proper sponsorship for your lib.

I think that these would be the right signals to make it easier for developers of larger projects to choose this library. 
And above all, it is easier to convince our superiors.

Regards

**Reply by vnbaaij** [reply]
> @vnbaaij thank you for your really fast response.
> 
> I would be more reassured if Microsoft invested in this project as well. :)

So would I 😀

> Does moving the project under the Microsoft domain (github) mean that this may happen in the future?

The project has always lived under the github.com/microsoft organization.  It living there does not mean there will be feature investments.  It does make it easier to do that in the future if decided so.

> I really don't understand how it is possible that a project like blazor doesn't have at least a basic library. or at least proper sponsorship for your lib.

No comment 😉

> I think that these would be the right signals to make it easier for developers of larger projects to choose this library. And above all, it is easier to convince our superiors.

Agreed!
